### PR TITLE
update SCM for adding CCPP dependencies to metadata and active attribute

### PR DESCRIPTION
**Pull request by grantfirl**
This PR follows https://github.com/NOAA-EMC/fv3atm/pull/153 in order to work with dependencies in the CCPP metadata and "active" attribute in the same.

Associated PRs:
https://github.com/NCAR/ccpp-framework/pull/317
https://github.com/NCAR/ccpp-physics/pull/483

This compiles/runs on Mac/GNU. Have not tested on other platforms.

### CODE REVIEW

**Comment by grantfirl** [comment]
Note: The https://github.com/NOAA-EMC/fv3atm/pull/153 PR does not appropriately apply "active" attribute for all diagnostic tendencies (du3dt, dv3dt, dt3dt, dq3dt). These WERE included in this SCM PR.

**Comment by climbfuji** [comment]
> Note: The [NOAA-EMC/fv3atm#153](https://github.com/NOAA-EMC/fv3atm/pull/153) PR does not appropriately apply "active" attribute for all diagnostic tendencies (du3dt, dv3dt, dt3dt, dq3dt). These WERE included in this SCM PR.

Can you add the missing pieces to your DDT fv3atm PR?

**Comment by grantfirl** [comment]
> > Note: The [NOAA-EMC/fv3atm#153](https://github.com/NOAA-EMC/fv3atm/pull/153) PR does not appropriately apply "active" attribute for all diagnostic tendencies (du3dt, dv3dt, dt3dt, dq3dt). These WERE included in this SCM PR.
> 
> Can you add the missing pieces to your DDT fv3atm PR?

Yes.

**Comment by grantfirl** [comment]
@climbfuji Done: https://github.com/NOAA-EMC/fv3atm/pull/162/commits/65800dcb9467200dd0d277225f792ffef907f51b

**Comment by climbfuji** [comment]
Thanks!

**Comment by grantfirl** [comment]
@climbfuji @llpcarson FYI, the master branch of SCM is "broken" until this PR is merged.

**Comment by climbfuji** [comment]
> @climbfuji @llpcarson FYI, the master branch of SCM is "broken" until this PR is merged.

Why?

**Comment by grantfirl** [comment]
> > @climbfuji @llpcarson FYI, the master branch of SCM is "broken" until this PR is merged.
> 
> Why?

Broken in the sense that the latest master of SCM doesn't work with the latest master of ccpp-physics/framework due to the merged ccpp-table-properties stuff.

**Comment by climbfuji** [comment]
> > > @climbfuji @llpcarson FYI, the master branch of SCM is "broken" until this PR is merged.
> > 
> > 
> > Why?
> 
> Broken in the sense that the latest master of SCM doesn't work with the latest master of ccpp-physics/framework due to the merged ccpp-table-properties stuff.

But that is ok. gmtb-scm master points to specific hashes of ccpp-framework and ccpp-physics and those work, right? Nobody should blindly update each submodule/repository to the head of develop, master, ... and expect it to work.

**Comment by grantfirl** [comment]
> > > > @climbfuji @llpcarson FYI, the master branch of SCM is "broken" until this PR is merged.
> > > 
> > > 
> > > Why?
> > 
> > 
> > Broken in the sense that the latest master of SCM doesn't work with the latest master of ccpp-physics/framework due to the merged ccpp-table-properties stuff.
> 
> But that is ok. gmtb-scm master points to specific hashes of ccpp-framework and ccpp-physics and those work, right? Nobody should blindly update each submodule/repository to the head of develop, master, ... and expect it to work.

👍  Agreed, and yet people do.

File: ccpp/config/ccpp_prebuild_config.py [hunk 1]
```diff
@@ -85,134 +87,6 @@
         },
     }
 
-# Add all physics scheme dependencies relative to basedir - note that the CCPP
```
Comment (climbfuji) [inline ccpp/config/ccpp_prebuild_config.py:88]: Getting rid of all of this is really nice.

**Review by climbfuji — APPROVED** [review]
Thanks for making those changes and getting up to speed with the new tables/attributes!

**Comment by grantfirl** [comment]
@climbfuji Thanks for leading the way with the fv3atm PR!

**Pull request merged**